lib/topological_inventory/orchestrator/event_manager.rb

```diff
@@ -103,6 +104,48 @@ def process_events(events)
         metrics&.record_error(:event_manager)
       end
 
+      # Check the (k8s|topo|sources) apis to make sure we're healthy
+      def health_checker
+        sources = URI.parse(worker.api.sources_api)
+        topology = URI.parse(worker.api.topology_api)
+        k8s = worker.send(:object_manager)
+
+        loop do
+          errors = []
+
+          errors << check_url(topology)
+          errors << check_url(sources)
+          errors << check_k8s(k8s)
+
+          if errors.any?
+            FileUtils.rm_f("/tmp/healthy")
+          else
+            FileUtils.touch("/tmp/healthy")
+          end
+
+          sleep 60
```

Comment:
Should this be managed by an env variable? If not is there a way for the deployment configuration to fall out of sync with this integer for sleep?

Reply:
Good point. I'll update the PR to have a template for the interval, that way one update will update them both and they're always in sync.